.gitmodules

```diff
@@ -1,3 +1,29 @@
 [submodule "third_party/prometheus-cpp"]
 	path = third_party/prometheus-cpp
-	url = https://github.com/jupp0r/prometheus-cpp.git
+	url = https://github.com/jupp0r/prometheus-cpp
+	branch = master
```

Comment:
Do we need create submodule based on stable branch instead of master?

Reply:
They are publishing tags on `master` , so we do use their stable commit associated with latest tag. When submodule checkout is happening, it is fetching from `master` + commit id. So we do pin to stable.